D1673/P1673.md

```diff
@@ -6147,57 +6199,17 @@ constexpr index_type stride(rank_type) const noexcept;
 
 [16]{.pnum} *Returns:* 1 if `extent(0)` is less than 2, else 0.
 
-## Scaled in-place transformation [linalg.scaled]
-
-### Introduction [linalg.scaled.intro]
-[1]{.pnum} The `scaled` function takes a value `alpha` and an `mdspan` `x`,
-and returns a new read-only `mdspan`
-that represents the elementwise product of `alpha` with each element of `x`.
-
-[*Example:*
-```c++
-using Vec = mdspan<double, dextents<size_t, 1>>;
-
-// z = alpha * x + y
-void z_equals_alpha_times_x_plus_y(
-  double alpha, Vec x,
-  Vec y,
-  Vec z)
-{
-  add(scaled(alpha, x), y, z);
-}
-
-// z = alpha * x + beta * y
-void z_equals_alpha_times_x_plus_beta_times_y(
-  double alpha, Vec x,
-  double beta, Vec y,
-  Vec z)
-{
-  add(scaled(alpha, x), scaled(beta, y), z);
-}
-```
---*end example*]
-
-<!--
-An implementation could dispatch to a function
-in the BLAS library, by noticing that the first argument
-has an `scaled_accessor` `Accessor` type.
-It could use this information to extract the appropriate
-run-time value(s) of the relevant BLAS function arguments
-(e.g., `ALPHA` and/or `BETA`),
-by calling `scaled_accessor::scaling_factor`.
--->
+### Exposition only helpers [linalg.helpers]
 
-### Exposition-only function object _`abs-if-needed`_ [linalg.scaled.abs]
+#### _`abs-if-needed`_ [linalg.helpers.abs]
 
 [1]{.pnum} The name _`abs-if-needed`_ denotes an exposition-only function object.
 The expression _`abs-if-needed`_`(E)` for subexpression `E` whose type is `T`
 is expression-equivalent to:
 
-* [1.1]{.pnum} `E`
-    if `value_type` is an unsigned integer;
+* [1.1]{.pnum} `E` if `value_type` is an unsigned integer; otherwise
```

Comment:
For consistency, should we consider attaching "otherwise" to the clause that explains what we do in that otherwise case?

```suggestion
* [1.1]{.pnum} `E` if `value_type` is an unsigned integer;
```

(and the change below)